tests/extensions/test_files_store.py

```diff
@@ -70,6 +73,30 @@ def test_spider_closed_even(caplog):
         ]
 
 
+def test_spider_closed_no_data(caplog):
+    spider = spider_with_files_store('no_data')
+
+    extension = FilesStore.from_crawler(spider.crawler)
+    with caplog.at_level(logging.INFO):
+        extension.spider_closed(spider, 'finished')
```

Comment:
Add newline between test setup and assertions, for easier visual skimming.

```suggestion
        extension.spider_closed(spider, 'finished')

```